test_scripts/RC/ButtonPress/005_check_rpc_paramters_for_button_press.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------------
+-- RPC: ButtonPress
+-- Script: 005
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local commonRC = require('test_scripts/RC/commonRC')
+local runner = require('user_modules/script_runner')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+
+--[[ Local variables for tests ]]
+local climate_button_names = {
+    "AC_MAX",
+    "AC",
+    "RECIRCULATE",
+    "FAN_UP",
+    "FAN_DOWN",
+    "TEMP_UP",
+    "TEMP_DOWN",
+    "DEFROST_MAX",
+    "DEFROST",
+    "DEFROST_REAR",
+    "UPPER_VENT",
+    "LOWER_VENT"
+}
+
+local radio_button_names = {
+    "VOLUME_UP",
+    "VOLUME_DOWN",
+    "EJECT",
+    "SOURCE",
+    "SHUFFLE",
+    "REPEAT",
+}
+
+local climate_params = {
+    moduleType = "CLIMATE",
+    buttonName = "AC_MAX",
+    buttonPressMode = "SHORT" -- LONG, SHORT
+}
+
+local radio_params = {
```

Comment:
accepted, corrected in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1574/commits/b4f76da576e577944521d50663b84a1780ffa2cb